### DISCUSSION TITLE
I am trying to use it in expo Project but its giving dependency error

### DISCUSSION THREAD

**Discussion by talha882** [react-native-modal/react-native-modal#625] [Q&A]
![modal](https://user-images.githubusercontent.com/89183221/140910585-195476b4-a2f9-46e5-a834-6bf32005b800.PNG)

**Comment by mmazzarolo** [comment]
@talha882 you must be on react-native@>=0.65.0 for `react-native-modal` to work

**Reply by talha882** [reply]
I am already on the required Reeact-native version.

**Reply by mmazzarolo** [reply]
🤔 
![image](https://user-images.githubusercontent.com/9536354/141770401-79a71861-3ff4-44c5-b0f1-2f9370456d93.png)
